### PR TITLE
Add versions for Safari wheel events

### PR DESCRIPTION
**Pull request by Cactusbone**
Also remove pinch to zoom support, since I could not get it to work
I used https://www.quirksmode.org/dom/events/tests/scroll.html and browserstack to test wheel event support

### CODE REVIEW

**Comment by Cactusbone** [comment]
Note concerning version:
Using browserstack, I was able to confirm version 6.0 of safari does not support wheel event, but 6.2 does. However, there is no 6.1 version available to test, so I cannot confirm if it works or not on 6.1

**Comment by Cactusbone** [comment]
sure thing for 6.1 :)

I could not get the wheel event to work on any Safari iOS, still using browserstack, however i do not have a bluetooth mouse to test it on a real device, so I'm not sure the event is available on Safari iOS

**Comment by vinyldarkscratch** [comment]
Oh yeah, derp — sorry, never mind the Safari iOS stuff!  I didn't read the PR carefully before submitting the review; iOS doesn’t have any mouse support*, thus we should actually set it to “false” instead.

* _iPadOS has support in an accessibility manner; that is, mouse events are translated into touch events by the system, so web apps wouldn’t have access to the mouse events._

**Comment by vinyldarkscratch** [comment]
(Oh by the way, we always squash merge our PRs, so no need to force push!)

Thanks for the changes! I will review them as soon as I get to my flight’s gate.

**Comment by Cactusbone** [comment]
Thanks for the fast review :)

File: api/WheelEvent.json [hunk 1]
```diff
@@ -320,10 +320,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": true
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/WheelEvent.json]: It looks like based upon your testing, this should be `false`?

Comment (Cactusbone) [reply api/WheelEvent.json]: indeed !

**Comment by Cactusbone** [comment]
Based on the "old" french version https://developer.mozilla.org/fr/docs/Web/API/Element/wheel_event,
It seems we lost some precision, most notably internet explorer does not handle "onwheel" attribute

**Comment by vinyldarkscratch** [comment]
The Safari changes look good to me!  I'll need to review the IE additions a little bit more -- I think it would be more reasonable to separate those changes into a separate PR, so we can keep the scope of this PR to Safari data, and thus can merge it right away?

**Comment by Cactusbone** [comment]
Yep, I'll move IE changes to another PR :)

**Pull request merged**